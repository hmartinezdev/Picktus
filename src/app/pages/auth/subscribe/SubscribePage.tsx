import SubscribeHandler from '@containers/SubscribeHandler';
import Authentication from '@layouts/Authentication';
import React from 'react';
import { IPageProps } from '../../page.type';

const SubscribePage = (props: IPageProps): React.ReactElement<Authentication> => (
  <Authentication {...props}>
    <SubscribeHandler />
  </Authentication>
);

SubscribePage.displayName = 'Auth';

export default SubscribePage;
