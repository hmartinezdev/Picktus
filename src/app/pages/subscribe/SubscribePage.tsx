import Subscribe from '@containers/Subscribe';
import React from 'react';
import { IPageProps } from '../page.type';

const SubscribePage = (props: IPageProps): React.ReactElement<SubscribePage> => <Subscribe {...props} />;

SubscribePage.displayName = 'SubscribePage';

export default SubscribePage;
