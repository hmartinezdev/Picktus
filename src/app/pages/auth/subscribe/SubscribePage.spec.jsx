import { shallow } from 'enzyme';
import React from 'react';
import SubscribePage from './SubscribePage';

let wrapper;

const setup = () => shallow(<SubscribePage />);

describe('<SubscribePage />', () => {
  test('should render properly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
