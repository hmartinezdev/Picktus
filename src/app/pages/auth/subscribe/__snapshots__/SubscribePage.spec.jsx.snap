// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubscribePage /> should render properly 1`] = `
<Authentication>
  <ConnectFunction
    key="SubscribeHandler"
  />
</Authentication>
`;
